model/user.go

```diff
@@ -10,6 +12,7 @@ type User struct {
 	EmailAddress string
 	Password []byte
 	UserRole      int
+	UserPasswordMd5	sql.NullString
```

Comment:
You can just do string here, I think? Any reason to use sql.NullString?

Reply:
I seemed to be getting an error here when scanning null into that field, but I'll play around with it a little bit. 

https://github.com/DarthHater/bored-board-service/blob/HandleLegacyPasswords/database/database.go#L143